// an-random/server.js
//Aqui los npm
const fetch = require("node-fetch");


//Archvivos para los GIFS
let hugs = require("./GIFS/hug.json");
let kiss = require("./GIFS/kiss.json");
let sleep = require("./GIFS/sleep.json");
let run = require("./GIFS/run.json");
let punch = require("./GIFS/punch.json");
let bored = require("./GIFS/bored.json");
let confused = require("./GIFS/confused.json")
let cry = require("./GIFS/cry.json");
let dance = require("./GIFS/dance.json")
let happy = require("./GIFS/happy.json");
let hi = require("./GIFS/hi.json");
let kill = require("./GIFS/kill.json");
let lick = require("./GIFS/lick.json");
let pat = require("./GIFS/pat.json");
let revive = require("./GIFS/revive.json")
let slap = require("./GIFS/slap.json")
let suicide = require("./GIFS/suicide.json")
let blush = require("./GIFS/blush.json")

//Aqui van las funciones
module.exports = {
  
  //Mira para crear una funcion es asi  ==> 
  getHug: async function() {
    let abrazos = hugs;
    let randomHug = abrazos[Math.floor(abrazos.length * Math.Random())]
    let resultadoHug = randomHug
    return resultadoHug;
  },
  
  getKiss: async function() {
    let besos = kiss;
    let randomKiss = besos[Math.floor(besos.length * Math.Random())]
    let resultadoKiss = randomKiss
    return resultadoKiss;
  },
  
  getSleep: async function() {
    let duerme_puto = sleep;
    let randomSleep = duerme_puto[Math.floor(duerme_puto.length * Math.Random())]
    let resultadoSleep = randomSleep
    return resultadoSleep;
  },
  
  getRun: async function() {
    let corre = run;
    let randomRun = corre[Math.floor(corre.length * Math.Random())]
    let resultadoRun = randomRun
    return resultadoRun;
  },
  
  getPunch: async function() {
    let golpear = punch;
    let randomPunch = golpear[Math.floor(golpear.length * Math.Random())]
    let resultadoPunch = randomPunch
    return resultadoPunch;
  },
  
  getBored: async function() {
    let aburrido = bored;
    let randomBored = aburrido[Math.floor(aburrido.length * Math.Random())]
    let resultadoBored = randomBored
    return resultadoBored;
  },
  
  getConfused: async function(){
    let confundido = confused;
    let randomConfused = confundido[Math.floor(confundido.length * Math.Random())]
    let resultadoConfused = randomConfused;
    return resultadoConfused;
  },
  
  getCry: async function() {
    let llorar = cry;
    let randomCry = llorar[Math.floor(llorar.length * Math.Random())]
    let resultadoCry = randomCry;
    return resultadoCry;
  },
  
  getDance: async function() {
    let bailar = dance;
    let randomDance = bailar[Math.floor(bailar.length * Math.Random())]
    let resultadoDance = randomDance;
    return resultadoDance;
  },
  
  getHappy: async function(){
    let feliz = happy
    let randomHappy = feliz[Math.florr(feliz.length * Math.Random())]
    let resultadoHappy = randomHappy
    return resultadoHappy;
  },
  
  getHi: async function() {
    let hola = hi;
    let randomHI = hola[Math.floor(hola.lenght * Math.Random())]
    let resultadoHi = randomHI
    return resultadoHi;
    
  },
  
  getKill: async function() {
    let matar = kill
    let randomKill = matar[Math.floor(matar.length * Math.Random())]
    let resultadoKill = randomKill
    return randomKill
  },
  getSuicide: async function(){
    let suicidar = suicide
    let randomSuicide = suicidar[Math.floor(suicidar.length * Math.Random())]
    let resultadoSuicide = randomSuicide
    return randomSuicide
  }

}
